test/gtest/uct/uct_test.h

```diff
@@ -432,13 +432,15 @@ class test_uct_iface_attrs : public uct_test {
     entity *m_e;
 };
 
-
-#define UCT_TEST_IB_TLS \
-    rc_mlx5,            \
-    rc_verbs,           \
-    dc_mlx5,            \
-    ud_verbs,           \
-    ud_mlx5,            \
+#define UCT_TEST_IB_MLX_TLS \
+    rc_mlx5,                \
+    dc_mlx5,                \
+    ud_mlx5
+
+#define UCT_TEST_IB_TLS  \
+    UCT_TEST_IB_MLX_TLS, \
+    rc_verbs,            \
```

Comment:
why need to separate? it affects other tests as well

Reply:
ok, I'll return back, but skip will not set AR_ENABLE for Verbs TLs
is it ok?